pkg/commenter/jenkins/jenkins.go

```diff
@@ -37,6 +40,32 @@ func NewJenkins(baseRef string) (commenter.Repository, error) {
 			}
 			return bitbucket_server.NewBitbucketServer(apiUrl, username, password, bitbucketutils.GetPrId(), project, repo, baseRef)
 		}
+	} else if strings.Contains(cloneUrl, "github") {
```

Comment:
This works for aqua but the check is not valid..
Argon has a GH-server with url `https://gh-staging.argon.io/login`.
We need to detect it in another way, or use go-environments